js/stat.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var CLOUD_WIDTH = 420;
+var CLOUD_HEIGHT = 270;
+var CLOUD_X = 100;
+var CLOUD_Y = 10;
+var CLOUD_OFFSET = 10;
+var COL_WIDTH = 40;
+var COL_MAX_HEIGHT = 150;
+var COL_X = 140;
+var COL_Y = CLOUD_HEIGHT - CLOUD_OFFSET - 20;
+var NAME_Y = CLOUD_HEIGHT - CLOUD_OFFSET;
+var MARGIN_COLS = 50;
+var FONT = '16px PT Mono';
+var FONT_COLOR = 'rgb(0, 0, 0)';
+
+var renderCloud = function (ctx, x, y, color) {
+  ctx.fillStyle = color;
+  return ctx.fillRect(x, y, CLOUD_WIDTH, CLOUD_HEIGHT);
+};
+
+var getMaxElement = function (array) {
+  if (array.length > 0) {
+    var maxElement = array[0];
+
+    for (var i = 0; i <= array.length - 1; i++) {
```

Comment:
у цикла `for` есть закрепленный паттерн написания его условия: 
```suggestion
    for (var i = 0; i < array.length ; i++) {
```